portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps.servlets;
+
+import java.io.Console;
+import java.io.IOException;
+import java.util.ArrayList;
+//import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/information")
+public final class InformationServlet extends HttpServlet
+{
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        //Strings
+        String currentInfo = "I'm taking finals this week";
+        String pastInfo = "I did a few art commisions this past month" ;
+        String futureInfo = "I'll be aplying to community service programs this summer";
+        //Create array of strings
+        ArrayList<String> personalInfo = new ArrayList<String>();
+        personalInfo.add(currentInfo);
+        personalInfo.add(pastInfo);
+        personalInfo.add(futureInfo);
+        String json = convertToJson(personalInfo);
```

Comment:
I think the correct parameter to gson.convertToJson is a Java Object(From a java class). A Json object has a "tag" and a "body", while an ArrayList element only has a "body" so they cannot be directly converted.
See **software-product-sprint/walkthroughs/week-2-server/examples/server-stats/src/main/java/com/google/sps/data/ServerStats.java** for a Java class that holds the data.

Reply:
Alright, I'll check this out.

Reply:
Worked like a charm, thanks Chen